pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+```mermaid
+sequenceDiagram
+    participant Requester as Requester
+    participant Responder as Responder
+    Requester->>Responder: Send Request
+    activate Responder
+    Responder-->>Requester: Send Reply
+    deactivate Responder
+```
+
+## Sub-patterns
+
+### Dynamic response Channel
+There are situations where you do not know the reply channel at the design time. Instead, the reply channel is dynamically determined at runtime, based on factors such as the request message payload or header. 
+
+In this case, since you don't know what the address of the reply channel is yet, you can either assign null to the `address` property or omit the property entirely indicating that the address is not known yet. You can now use `Operation Reply Address` object to dynamically specify where the reply has to be sent. The `Operation Reply Address` object has a property called `location` which essentially takes in a runtime expression that specifies the address of the reply channel.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example for dynamic reply channel sub-pattern
+  version: 1.0.0
+  description: Example with a requester that initiates the request/reply pattern where the reply will happen on whatever is defined in the header `replyTo` of the request.
+
+channels:
+  ping:
+    address: /ping
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: null
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      address:
+        description: The reply address is dynamically determined based on the request header `replyTo`
+        location: "$message.header#/replyTo"
+      channel: 
+        $ref: '#/channels/pong'
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      address:
+        description: The reply address is dynamically determined based on the request header `replyTo`
+        location: "$message.header#/replyTo"
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+### Request/reply over the same channel
+The request/reply pattern can also be implemented over the same channel. You can do this by specifying the same address for both the requester and the replier.
+
+Here's a example of setting up both requestor and replier in the same channel:
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with requester
+  version: 1.0.0
+  description: Requester example initiating the request-reply pattern.
+
+channels:
+  ping:
+    address: /
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: /
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+  pongReply:
```

Comment:
same as with other example, this app do not receive pong by default, and this should be mentioned only under reply of pingRequest